src/currentprocess/filesource.rs

```diff
@@ -1,11 +1,15 @@
-use std::io::{self, BufRead, Cursor, Read, Result, Write};
-use std::sync::{Arc, Mutex, MutexGuard};
+use std::io::{self, BufRead, Read, Result, Write};
 
 use enum_dispatch::enum_dispatch;
 
+use super::terminalsource::{ColorableTerminal, StreamSelector};
 use crate::currentprocess::process;
 
-use super::terminalsource::{ColorableTerminal, StreamSelector};
+#[cfg(feature = "test")]
+use std::{
```

Comment:
Prefer to keep this next to the other std imports (in the same block).

Reply:
Done.